libs/net/dal/Migrations/1.0.12/Up/PostUp/00-NotificationRazorTemplate.sql

```diff
@@ -0,0 +1,78 @@
+DO $$
+BEGIN
+
+UPDATE public."notification"
+SET
+	settings=to_json(regexp_replace('{ "subject": "
```

Comment:
Add the various URI values in the settings.

Reply:
Updated, thanks